spec/components/validators/integer_setting_validator_spec.rb

```diff
@@ -21,9 +21,21 @@
 
       it "returns true if value is a valid integer" do
         expect(validator.valid_value?(1)).to eq(true)
-        expect(validator.valid_value?(-1)).to eq(true)
         expect(validator.valid_value?('1')).to eq(true)
-        expect(validator.valid_value?('-1')).to eq(true)
+      end
+
+      it "defaults min to 0" do
```

Comment:
This description is alittle odd for me because the test cases aren't really testing if the min is "0". The default min could be 5 and the test cases would still pass.